skore/src/skore/cli/cli.py

```diff
@@ -9,7 +9,7 @@
 from skore.cli.quickstart_command import __quickstart
 
 
-def cli(args: list[str]):
```

Comment:
Is there an advantage to this global variable approach? Despite how `parser.parse_args` behaves, I'd rather functions be as pure as possible

Reply:
Actually this is something back from when I moved in `click`. `click` will use decorator for that. I think that I can revert it in this case.